README.rst

```diff
@@ -33,24 +33,14 @@ found in the `Generating support keys`_.
 Top_
 
 
-API Overview
-============
-
-To show the help of service, you can use the operation::
-
-    GET /    (text/html)
+API Reference Documentation
+---------------------------
 
-To query the keys, you should use the next operations::
-
-    GET /v1/support/<region_name>/sshkey  (text/plain)
-    
-    GET /v1/support/<region_name>/gpgkey  (text/plain)
+- `FIWARE Aiakos v1 (Apiary)`__
 
-To upload the keys, you should use the next operation, adding payload with the key::
+__ `FIWARE Aiakos - Apiary`_
 
-    POST /v1/support/ (text/plain)
 
-    (Use X-Auth-Token in header)
 Top_
 
```

Comment:
You have to take a look to the documentation of https://github.com/telefonicaid/fiware-cloto#api-overview

In the section API Overview, you should include a basic example of the use of the API of the component and make reference to the subsection 'API Reference Documentation' in which you put the link like you have right now.

Bear in mind that it is not necessary to make reference to the text "...and at the user and programmer guide for more description of the possibilities and operations." due to be do not have user and programmers guide of this component.